core/src/main/resources/i18n/displayStrings.properties

```
@@ -2659,10 +2659,6 @@ popup.privateNotification.headline=Important private notification!
 popup.securityRecommendation.headline=Important security recommendation
 popup.securityRecommendation.msg=We would like to remind you to consider using password protection for your wallet if you have not already enabled that.\n\nIt is also highly recommended to write down the wallet seed words. Those seed words are like a master password for recovering your Bitcoin wallet.\nAt the \"Wallet Seed\" section you find more information.\n\nAdditionally you should backup the complete application data folder at the \"Backup\" section.
 
-popup.warning.localNodeMisconfigured.explanation=Bisq detected a locally running Bitcoin Core node (at localhost); however, its configuration is incompatible with Bisq.\n\n\
-For Bisq to use a local Bitcoin node, the node has to have pruning disabled and bloom filters enabled. Starting with Bitcoin Core v0.19 you have to manually enable bloom filters by setting peerbloomfilters=1 in your bitcoin.conf configuration file.
-popup.warning.localNodeMisconfigured.continueWithoutLocalNode=Continue without using local node
-
 popup.bitcoinLocalhostNode.msg=Bisq detected a locally running Bitcoin Core node (at localhost) and is using it.\n\
   Please make sure that this node is fully synced before you start Bisq.
 popup.bitcoinLocalhostNode.additionalRequirements=\n\nThe node was found to be well configured. For reference, the requirements are for the node to have pruning disabled and bloom filters enabled.
```

Should this message remain without the proper check?

---

Didn't notice that. Should be changed. Have to go out for an hour; will change it when I come back.

---

Good catch - this should be in that case
```suggestion
popup.bitcoinLocalhostNode.additionalRequirements=\n\nStarting with Bitcoin Core v0.19 you have to manually enable bloom filters by setting peerbloomfilters=1 in your bitcoin.conf configuration file.
```